clients/admin-ui/src/features/config-wizard/organization.slice.ts

```diff
@@ -0,0 +1,143 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import { HYDRATE } from "next-redux-wrapper";
+
+import type { AppState } from "../../app/store";
+import {
+  Organization,
+  OrganizationParams,
+  OrganizationResponse,
+} from "./types";
+
+export interface State {
+  page: number;
+  size: number;
+  token: string | null;
+}
+
+const initialState: State = {
+  page: 1,
+  size: 25,
+  token: null,
+};
+
+// Helpers
+export const mapFiltersToSearchParams = ({
+  page,
+  size,
+}: Partial<OrganizationParams>) => ({
+  ...(page ? { page: `${page}` } : {}),
+  ...(typeof size !== "undefined" ? { size: `${size}` } : {}),
+});
+
+// Organization API
+export const organizationApi = createApi({
+  reducerPath: "organizationApi",
+  baseQuery: fetchBaseQuery({
+    baseUrl: process.env.NEXT_PUBLIC_FIDESCTL_API,
+    prepareHeaders: (headers, { getState }) => {
+      const { token } = (getState() as AppState).user;
+      headers.set("Access-Control-Allow-Origin", "*");
```

Comment:
we shouldn't need this on the fidesctl side, see https://github.com/ethyca/fides/pull/686#discussion_r881926662